### PR TITLE
Bugfix/error updates

### PR DESCRIPTION
**Pull request by piyalbasu**
Screen Shot 2020-11-04 at 3 17 53 PM

### CODE REVIEW

File: extension/src/popup/helpers/validators.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const confirmPassword = YupString()
 
 export const termsOfUse = YupBool().oneOf(
   [true],
-  "Agreeing to the Terms of Use is required",
+  "Agreeing to the Terms of Use and the Privacy Policy is required",
```
Comment (piyalbasu) [inline extension/src/popup/helpers/validators.tsx:16]: fix terms and conditions error

File: extension/src/popup/views/SignTransaction.tsx [hunk 2]
```diff
@@ -114,21 +119,46 @@ const SubmitButtonEl = styled(SubmitButton)`
   width: 12.43rem;
 `;
 
+const NetworkMismatchWarning = () => (
```
Comment (piyalbasu) [inline extension/src/popup/views/SignTransaction.tsx:122]: and network mismatch error

**Review by jeesunikim — APPROVED** [review]

File: @shared/constants/stellar.ts [hunk 3]
```diff
@@ -1,11 +1,20 @@
...
+
+export const OTHER_NETWORK_NAME = !isTestnet ? TESTNET : PUBNET;
 
 export const NETWORK_URL = isTestnet
   ? "https://horizon-testnet.stellar.org"
   : "https://horizon.stellar.org";
+
+export const NETWORK_PASSPHRASE = isTestnet
+  ? "Test SDF Network ; September 2015"
+  : "Public Global Stellar Network ; September 2015";
```
Comment (morleyzhi) [inline @shared/constants/stellar.ts:20]: I think we should get these from the Stellar SDK (`StellarSdk.Networks.TESTNET` / `PUBLIC`  )

File: extension/src/popup/views/SignTransaction.tsx [hunk 4]
```diff
@@ -114,21 +119,46 @@ const SubmitButtonEl = styled(SubmitButton)`
...
+  <>
+    <WarningMessage subheader={`Freighter is currently on ${NETWORK_NAME}`}>
+      <p>The transaction you're trying to sign is on {OTHER_NETWORK_NAME}.</p>
```
Comment (morleyzhi) [inline extension/src/popup/views/SignTransaction.tsx:125]: You used a single quote in `you're` where you probably meant to use an apostrophe `you’re`. See https://github.com/stellar/product-conventions/blob/master/STYLEGUIDE.md#use-the-right-symbols.

**Review by morleyzhi — APPROVED** [review]
Overall approach looks good, just some small details.

**Pull request merged**